payouts/payer.go

```diff
@@ -8,14 +8,14 @@ import (
 	"strconv"
 	"time"
 
-	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/dominant-strategies/go-quai/common/hexutil"
 
-	"github.com/J-A-M-P-S/go-etcstratum/rpc"
-	"github.com/J-A-M-P-S/go-etcstratum/storage"
-	"github.com/J-A-M-P-S/go-etcstratum/util"
+	"github.com/dominant-strategies/go-quai-stratum/rpc"
+	"github.com/dominant-strategies/go-quai-stratum/storage"
+	"github.com/dominant-strategies/go-quai-stratum/util"
 )
 
-const txCheckInterval = 5 * time.Second
+const txCheckInterval = 5*time.Secondominant - dominant - strategies
```

Comment:
what is this? Copy paste mistakes?

Reply:
Wow! I guess because this file isn't used Go doesn't bother to compile it.